### ISSUE TITLE
Release version on Device can not run.

### ISSUE THREAD

**Issue by steve3d** [juliansteenbakker/mobile_scanner#191]
I have a project using `qr_code_scanner`, now I'm upgrading to this, and debug version works fine. But the release version can't.

I have these errors, then the app just crashed.

```
E/AndroidRuntime(23097): java.lang.ExceptionInInitializerError
E/AndroidRuntime(23097):        at androidx.camera.camera2.internal.Camera2CameraControlImpl.<init>(Camera2CameraControlImpl.java:29)
E/AndroidRuntime(23097):        at androidx.camera.camera2.internal.u.<init>(Camera2CameraImpl.java:24)
E/AndroidRuntime(23097):        at androidx.camera.camera2.internal.Camera2CameraFactory.getCamera(Camera2CameraFactory.java:5)
E/AndroidRuntime(23097):        at f.F.b(CameraRepository.java:5)
E/AndroidRuntime(23097):        at androidx.camera.core.z.b(CameraX.java:21)
E/AndroidRuntime(23097):        at androidx.camera.core.w.run(Unknown Source:10)
E/AndroidRuntime(23097):        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/AndroidRuntime(23097):        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/AndroidRuntime(23097):        at java.lang.Thread.run(Thread.java:920)
E/AndroidRuntime(23097): Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodException: f.r.values []
E/AndroidRuntime(23097):        at java.lang.Enum.enumValues(Enum.java:270)
E/AndroidRuntime(23097):        at java.lang.Enum.access$000(Enum.java:61)
E/AndroidRuntime(23097):        at java.lang.Enum$1.create(Enum.java:277)
E/AndroidRuntime(23097):        at java.lang.Enum$1.create(Enum.java:275)
E/AndroidRuntime(23097):        at libcore.util.BasicLruCache.get(BasicLruCache.java:63)
E/AndroidRuntime(23097):        at java.lang.Enum.getSharedConstants(Enum.java:289)
E/AndroidRuntime(23097):        at java.lang.Class.getEnumConstantsShared(Class.java:2459)
E/AndroidRuntime(23097):        at java.util.EnumSet.getUniverse(EnumSet.java:407)
E/AndroidRuntime(23097):        at java.util.EnumSet.noneOf(EnumSet.java:109)
E/AndroidRuntime(23097):        at java.util.EnumSet.of(EnumSet.java:283)
E/AndroidRuntime(23097):        at androidx.camera.camera2.internal.B.<clinit>(Camera2CapturePipeline.java:2)
E/AndroidRuntime(23097):        ... 9 more
E/AndroidRuntime(23097): Caused by: java.lang.NoSuchMethodException: f.r.values []
E/AndroidRuntime(23097):        at java.lang.Class.getMethod(Class.java:2103)
E/AndroidRuntime(23097):        at java.lang.Class.getDeclaredMethod(Class.java:2081)
E/AndroidRuntime(23097):        at java.lang.Enum.enumValues(Enum.java:267)
E/AndroidRuntime(23097):        ... 19 more
```

then I try to create an empty project which only use this package. then both debug/release version works. on iOS, everything works.

I'm not an android expert, so I need help about these error. Thanks.

**Comment by minhtritc97** [comment]
same issue

**Comment by steve3d** [comment]
well, seems author doesn't care, or unable to solve this, I've successfully switched to more like a official solution by using [camera](https://pub.dev/packages/camera) and [google_mlkit_barcode_scanning](https://pub.dev/packages/google_mlkit_barcode_scanning) to solve the scan.

and finally I had tap to focus function.

**Comment by minhtritc97** [comment]
resolve by import mobile_scanner: 1.1.0 and wait for new version

**Comment by juliansteenbakker** [comment]
This issue is probably fixed with #257. I will release this as version 3.0.0 after i'm done with a few other functions.
